src/components/homePage/SchedulePreview.vue

```diff
@@ -13,4 +51,94 @@ export default {
 </script>
 
 <style scoped>
+.schedule-link {
+	color: var(--main-color);
+	text-decoration: underline;
+}
+.header {
+	display: flex;
+	align-items: baseline;
+	column-gap: 40px;
+}
+.content{
+	display: grid;
+	grid-template-columns: 40% 55%;
+	grid-auto-rows: 50% 50%;
+	grid-gap: 0% 4%;
+	grid-template-areas: 
+		"pink blue"
+		"pink green";
+}
+.box{
+	border-radius: 10px;
+	display: flex;
+}
```

Comment:
Only thing I would suggest is having `height: max-content;` so that the bottoms of the boxes snap to the content nicely.